include/SugarTheme/SugarTheme.php

```diff
@@ -965,12 +965,21 @@ protected function _getImageFileName(
      * @param  bool   $returnURL if true, returns URL with unique image mark, otherwise returns path to the file
      * @return string path of css file to include
      */
-    public function getCSSURL($cssFileName, $returnURL = true)
+    public function getCSSURL($cssFileName, $returnURL = true, $skipSubTheme = FALSE)
```

Comment:
I can not find any usages of third parameter `$skipSubTheme`. Is this function still works properly if I call it and pass `true` as `$skipSubTheme` value? Can I Test is some way? I think this function needs a unit and/or acceptance test to see it works properly in each calling situation.
Also: `FALSE` should be lowercase as it is in PSR standards what we use: 
https://www.php-fig.org/psr/psr-2/#25-keywords-and-truefalsenull

Reply:
@samus-aran Sorry, I was not specific enough. I meant I can not find a single call in the codebase something like this:
```
$theme->getCSSURL($cssFile, $retUrl, $thirdParameterHere); // "usages of third parameter"
```